README.md

```diff
@@ -70,13 +71,21 @@ environments etc etc!
 that you can spend most of your time developing agents.
 `rlberry` also provides implementations of several RL agents, benchmark environments and many other useful tools.
 
-Check our [getting started section](#getting-started)!
+## Installation
+
+Install the latest version for a stable release.
+
+```bash
+$ pip install -U git+https://github.com/rlberry-py/rlberry.git@v0.2.1#egg=rlberry[default]
+```
+
+rlberry dependencies are listed in [requirements.txt](https://github.com/rlberry-py/rlberry/blob/main/requirements.txt). The documentation includes more [installation instructions](https://rlberry.readthedocs.io/en/latest/installation.html) in particular for users that work with Jax.
```

Comment:
The dependencies depend on the installation mode, right? The [default] mode does not include pytorch, for example, but torch is in `requirements.txt`